book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
`rentals` should also have a setter. For that kindly use `attr_accessor` instead of `attr_reader`.

Reply:
Thanks for the kind feedback. Resolved